src/model/differ.js

```diff
@@ -119,6 +119,10 @@ export default class Differ {
 	 * @param {module:engine/model/operation/operation~Operation} operation An operation to buffer.
 	 */
 	bufferOperation( operation ) {
+		// Below we take an operation, check its type, then use its parameters in marking (private) methods.
+		// The general rule is to not mark elements inside inserted element. All inserted elements are re-rendered.
+		// Marking changes in them would cause a "double" changing then.
+		//
```

Comment:
Empty line.

Reply:
I do a lot of those to make comments more readable and separated from code, to make more breathing room so it is easier to read it all. These are not by mistake.